src/Uno.UI/UI/Xaml/Media/Animation/RepeatBehavior.cs

```diff
@@ -32,11 +32,11 @@ public RepeatBehavior(TimeSpan duration)
 			Count = 0;
 		}
 
-		public double Count { get; set; }
+		public double Count;
 
-		public TimeSpan Duration { get; set; }
+		public TimeSpan Duration;
 
-		public RepeatBehaviorType Type { get; set; }
+		public RepeatBehaviorType Type;
```

Comment:
Should we disable the code analysis checks for public fields to avoid these warnings?
![image](https://user-images.githubusercontent.com/1614057/188783654-e133a73b-4df8-4db9-af12-991b61f13e93.png)